### DISCUSSION TITLE
Docblocks for method

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#8257] [Ideas]
I noticed the lack of docblocks in Filament. I won't add them all in one shot, but rather, progressively as I encounter and use them.

Will PRs be accepted?

**Comment by danharrin** [comment]
It depends, if you're just going to repeat the method name and parameters then I'm not gonna add them. If there is actually some ambiguity then I'll accept PRs.

**Reply by bilogic** [reply]
Certainly not repetitive, but it doesn't have to cross the line of being ambiguous either.
I'm referring to the kind like Laravel, with the main purpose to help lessen the need to memorize the exact utility of a method and also to help those who are new to the library

```PHP
    /**
     * Return the remainder of a string after the first occurrence of a given value.
     *
     * @param  string  $subject
     * @param  string  $search
     * @return string
     */
    public static function after($subject, $search)
    {
        return $search === '' ? $subject : array_reverse(explode($search, $subject, 2))[0];
    }
```

**Reply by danharrin** [reply]
We always use native PHP types, so please don't include any `@param` or `@return` unless they use generics or add extra info

**Reply by bilogic** [reply]
Ok noted, let me collect a few before I file a PR thanks!
